roles/openshift_logging/tasks/install_kibana.yaml

```diff
@@ -15,8 +15,8 @@
     component: kibana-ops
     logging_component: kibana
     deploy_name: "logging-{{component}}"
-    image: "{{image_prefix}}logging-kibana:{{image_version}}"
-    proxy_image: "{{image_prefix}}logging-auth-proxy:{{image_version}}"
-    es_host: logging-es-ops
-  when: logging_use_ops
+    image: "{{openshift_logging_image_prefix}}logging-kibana:{{openshift_logging_image_version}}"
+    proxy_image: "{{openshift_logging_image_prefix}}logging-auth-proxy:{{openshift_logging_image_version}}"
+    openshift_logging_es_host: logging-es-ops
```

Comment:
I think this should stay 'es_host' in this case (and should be mirrored for the non-ops) so that we aren't changing the value of openshift_logging_es_host for all tasks that may follow this one.